lib/matplotlib/cbook.py

```diff
@@ -2445,6 +2445,14 @@ def safe_first_element(obj):
     return next(iter(obj))
 
 
+def asSequence(data):
```

Comment:
Hmm...while I agree, is the plan to limit it strictly to views? and then it might be unclear why non-views are being passed in.